### PR TITLE
[OPPRO-324]Add date type validation

### PR DESCRIPTION
**Pull request by JkSelf**
## What changes were proposed in this pull request?

(Please fill in changes proposed in this fix)

## How was this patch tested?

(Please explain how this patch was tested. E.g. unit tests, integration tests, manual tests)

(If this patch involves UI changes, please attach a screenshot; otherwise, remove this)

### CODE REVIEW

**Comment by JkSelf** [comment]
@rui-mo 
Please help to review. Thanks for your help!

**Comment by JkSelf** [comment]
@rui-mo Do you have any further comments?

**Comment by rui-mo** [comment]
> @rui-mo Do you have any further comments?

LGTM.

**Review by rui-mo — APPROVED** [review]

File: ep/build-velox/src/build_velox.sh [hunk 1]
```diff
@@ -14,8 +14,8 @@ BUILD_FOLLY=ON
 VELOX_BUILD_TYPE=release
 VELOX_HOME=/root/velox
 
-VELOX_REPO=https://github.com/oap-project/velox.git
-VELOX_BRANCH=main
+VELOX_REPO=https://github.com/JkSelf/velox.git
+VELOX_BRANCH=project-date-type
```
Comment (rui-mo) [inline ep/build-velox/src/build_velox.sh:18]: Should be changed back.

Comment (JkSelf) [reply ep/build-velox/src/build_velox.sh]: Done

**Pull request merged**